rpm/qtbase.spec

```diff
@@ -799,13 +808,15 @@ install -D -p -m 0644 %{_sourcedir}/qt.conf %{buildroot}%{_libdir}/qt5/bin/qt.co
 %{_libdir}/pkgconfig/Qt5Widgets.pc
 %{_datadir}/qt5/mkspecs/modules/qt_lib_widgets.pri
 %{_datadir}/qt5/mkspecs/modules/qt_lib_widgets_private.pri
+%{_libdir}/cmake/Qt5Widgets
 
 %files qtplatformsupport-devel
 %defattr(-,root,root,-)
 %{_includedir}/qt5/QtPlatformSupport/
 %{_libdir}/libQt5PlatformSupport.prl
 %{_libdir}/libQt5PlatformSupport.a
 %{_datadir}/qt5/mkspecs/modules/qt_lib_platformsupport_private.pri
+%{_libdir}/cmake/Qt5Core
```

Comment:
Is this intentionally the same that is already included in qtcore-devel?

Reply:
Duh, no, this didn't actually have cmake files.